Proc macros are broken on nightly 1.27

```
   Compiling tarpc-plugins v0.2.0
     Running `rustc --crate-name tarpc_plugins /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/tarpc-plugins-0.2.0/src/lib.rs --crate-type dylib --emit=dep-info,link -C prefer-dynamic -C debuginfo=2 -C metadata=74cc6fc8403b40c1 -C extra-filename=-74cc6fc8403b40c1 --out-dir /home/travis/build/mark-i-m/zippybook/code/client/target/debug/deps -L dependency=/home/travis/build/mark-i-m/zippybook/code/client/target/debug/deps --extern itertools=/home/travis/build/mark-i-m/zippybook/code/client/target/debug/deps/libitertools-40709c70d42af3b6.rlib --cap-lints allow`
error[E0609]: no field `identifier` on type `&mut syntax::ast::PathSegment`
   --> /home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/tarpc-plugins-0.2.0/src/lib.rs:129:23
    |
129 |                      .identifier);
    |                       ^^^^^^^^^^
```

```
rustc 1.27.0-nightly (eeea94c11 2018-04-06)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "tarpc"
-version = "0.10.0"
+version = "0.11.0"
 authors = ["Adam Wright <adam.austin.wright@gmail.com>", "Tim Kuehn <timothy.j.kuehn@gmail.com>"]
 license = "MIT"
 documentation = "https://docs.rs/tarpc"

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ arguments to tarpc fns.
 Add to your `Cargo.toml` dependencies:
 
 ```toml
-tarpc = "0.10.0"
+tarpc = "0.11.0"
 tarpc-plugins = "0.2.0"
 ```
 

--- a/RELEASES.md
+++ b/RELEASES.md
@@ -1,3 +1,8 @@
+## 0.10.0 (2018-04-08)
+
+## Breaking Changes
+Fixed rustc breakage in tarpc-plugins. These changes require a recent version of rustc.
+
 ## 0.10.0 (2018-03-26)
 
 ## Breaking Changes

--- a/examples/concurrency.rs
+++ b/examples/concurrency.rs
@@ -3,7 +3,7 @@
 // Licensed under the MIT License, <LICENSE or http://opensource.org/licenses/MIT>.
 // This file may not be copied, modified, or distributed except according to those terms.
 
-#![feature(conservative_impl_trait, plugin, never_type, use_extern_macros)]
+#![feature(plugin, never_type, use_extern_macros)]
 #![plugin(tarpc_plugins)]
 
 extern crate chrono;

--- a/rustfmt.toml
+++ b/rustfmt.toml
@@ -1,2 +1,1 @@
-ideal_width = 100
 reorder_imports = true

--- a/src/macros.rs
+++ b/src/macros.rs
@@ -634,7 +634,7 @@ mod functional_test {
                         let buf = include_bytes!("../test/root-ca.der");
                         let cert = unwrap!(Certificate::from_der(buf));
                         let mut connector = unwrap!(TlsConnector::builder());
-                        connector.add_root_certificate(cert);
+                        connector.add_root_certificate(cert).unwrap();
 
                         Context {
                             domain: DOMAIN.into(),

--- a/src/plugins/src/lib.rs
+++ b/src/plugins/src/lib.rs
@@ -126,7 +126,7 @@ fn ty_snake_to_camel(cx: &mut ExtCtxt, sp: Span, tts: &[TokenTree]) -> Box<MacRe
     convert(&mut path.segments
                      .last_mut()
                      .unwrap()
-                     .identifier);
+                     .ident);
     MacEager::ty(P(Ty {
         id: ast::DUMMY_NODE_ID,
         node: TyKind::Path(None, path),

